kaggle_environments/main.py

```diff
@@ -172,15 +233,25 @@ def action_handler(args):
             return action_step(args)
         elif args.action == "run":
             return action_run(args)
-        elif args.action == "load":
-            return action_load(args)
         else:
             return {"error": "Unknown Action"}
     except Exception as e:
         return {"error": str(e), "trace": traceback.format_exc()}
 
 
+log_path = None
```

Comment:
log_path is stored in a global so that if the http-server sets the log path on start, then subsequent invocations of act can reuse the same log_path.